pegasus/sites.v3/code.org/public/music/index.haml

```diff
@@ -3,42 +3,59 @@ title: Music Lab
 theme: responsive_full_width
 ---
 
+-# TODO ACQ-2556 - Clean this up after the 2024 HOC launch
+- hoc_nov_2024 = DCDO.get('hoc-2024-nov-launch', false)
+
 %link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
 %link{href: "/css/generated/page/music-styles.css", rel: "stylesheet"}
 
 %section.hero-banner-basic.overlay
   .wrapper.grid-container.justify-space-between.align-items-center.gap-4
     .text-wrapper
-      %p.body-one.white.has-icon
-        =hoc_s("music_lab.introducing_music_lab_strong", markdown: :inline, locals:{music_lab: "Music Lab"})
-      %h1.white
-        =hoc_s("music_lab.banner.heading")
-      %p.body-one.white
-        =hoc_s("music_lab.banner.desc")
-      %a.link-button.white{href: CDO.studio_url("/s/music-intro-2024/reset")}
-        =hoc_s("music_lab.button.try_music_lab")
+      -if hoc_nov_2024
+        %p.body-one.white.has-icon
+          =hoc_s("social_hoc_is_coming")
+        %h1.white
+          =hoc_s("music_lab.jam_session.name")
+        %p.body-one.white
+          =hoc_s("music_lab.jam_session.banner.desc_01")
+        %a.link-button.white{href: CDO.studio_url("/s/music-jam-2024/reset")}
+          =hoc_s("music_lab.jam_session.button.start")
+      -else
+        %p.body-one.white.has-icon
+          =hoc_s("music_lab.introducing_music_lab_strong", markdown: :inline, locals:{music_lab: "Music Lab"})
+        %h1.white
+          =hoc_s("music_lab.banner.heading")
+        %p.body-one.white
+          =hoc_s("music_lab.banner.desc")
+        %a.link-button.white{href: CDO.studio_url("/s/music-intro-2024/reset")}
+          =hoc_s("music_lab.button.try_music_lab")
+
       .afe.flex-container.align-items-baseline.wrap
         %p.add-margin-top.no-margin-bottom.white
           =hoc_s("music_lab.in_partnership_with")
         %figure
           %img{src: "/images/avatars/amazon_future_engineer_light.png", alt: hoc_s("music_lab.amazon_future_engineer")}
     %figure
-      %img{src: "/images/music-lab/music-lab-banner-graphic.png", alt: "", style: "width: 100%"}
+      %img{src: hoc_nov_2024 ? "/images/music-lab/music-lab-banner-graphic-best-of-stem.png" : "/images/music-lab/music-lab-banner-graphic.png", alt: "", style: "width: 100%"}
 
 %section.bg-neutral-light
   .wrapper.flex-container.justify-space-between.align-items-center.gap-1.wrap.flex-direction-column-mobile
     .text-wrapper.flex-1
       %h2.heading-lg
         =hoc_s("music_lab.featuring_songs_by")
       %p.no-margin-bottom
-        =hoc_s("music_lab.artist_list", markdown: :inline, locals:{artist_list: "The Chainsmokers, Tinashe, Rosa Linn, Coldplay, Aloe Blacc, Chloe x Halle, Lady Gaga, Dove Cameron"})
+        =hoc_s("music_lab.artist_list", markdown: :inline, locals:{artist_list: "Sabrina Carpenter, Shakira, Benson Boone, Henry Moodie, The Chainsmokers, Rosa Linn, Tinashe, Dove Cameron, Aloe Blacc"})
```

Comment:
This needs the DCDO logic to show for the launch and not before.

Reply:
ah, thank you!

Reply:
Can you update the "Without flag" screenshot up at the top too? It's still showing the Sabrina list.

Reply:
done!